mobify/platform-scaffold
# [Integration manager] merlins checkout shipping

This PR updates some of the checkout shipping commands for Merlins in the connector. This template was already partially moved into the connector, but this PR cleans up a few parts to make the demandware connector work easier. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1307
 **Linked PRs**: n/a

## Changes
- Moved checkout shipping commands into the checkout folder of the connector
- Moved UI fail/success actions back to the app and throw errors in connector commands that fail

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to the cart
- navigate to the checkout shipping page
- confirm the page still works as expected
